purchases/src/main/kotlin/com/revenuecat/purchases/common/LocaleProvider.kt

```diff
@@ -7,6 +7,29 @@ internal interface LocaleProvider {
 }
 
 internal class DefaultLocaleProvider : LocaleProvider {
+    private var orchestratorProvider: (() -> com.revenuecat.purchases.PurchasesOrchestrator?)? = null
+
+    fun setOrchestratorProvider(provider: () -> com.revenuecat.purchases.PurchasesOrchestrator?) {
```

Comment:
Should import instead of using fully qualified names (a few occurrences of this in this PR)